### ISSUE TITLE
508-defect-3 [SEMANTIC MARKUP, COGNITION]: The introtext SHOULD be a paragraph, not an H2

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#13558]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

While testing with screen readers, the introtext was introduced as a heading level 2. This text **should** be a paragraph with a class of `.va-introtext`. For screen reader users, it is confusing for this long passage of text to be a heading, as they may be navigating the page by headings and this is significantly longer than headings usually are. It is also inconsistent with the semantic markup used for intro text elsewhere on VA.gov.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a screen reader user, I want to consistently navigate the website by headings to understand the content hierarchy.

## Environment

* Operating System: all
* Browser: all
* Screenreading device: any
* Server destination: staging

## Steps to Recreate

1. Enter https://staging.va.gov/manage-va-debt/your-debt/ and please use Greg Anderson (user 1).
2. Start screenreading device listed in Environment
3. Verify that the following text is introduced as a heading level 2:

> Check the details of VA debt you might have related to your education, disability compensation, or pension benefits. Find out how to pay your debt and what to do if you need financial assistance.

### Current code

```
  <h2 class="vads-u-font-size--h3 vads-u-margin-top--0 vads-u-margin-bottom--2 vads-u-font-weight--normal vads-u-line-height--6">Check the details of VA debt you might have related to your education, disability compensation, or pension benefits. Find out how to pay your debt and what to do if you need financial assistance.</h2>
```

### Potential fix

```
<p class="va-introtext">Check the details of VA debt you might have related to your education, disability compensation, or pension benefits. Find out how to pay your debt and what to do if you need financial assistance.</p>
```

## WCAG or Vendor Guidance (optional)

* "The page MUST have meaningful headings to label each major section, which SHOULD start with `<h1>` (at the beginning of the main content, or at the beginning of every section of aggregated content, or at the beginning of modal dialogs), and SHOULD NOT skip heading levels, to allow screen reader users to navigate the tree structure of the heading hierarchy." [2.4.6](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/resources) By using an H2 here, it impacts the meaningfulness and utility of the page's heading architecture.

* "The text SHOULD be written in the simplest and clearest language possible, appropriate to the content." COG - https://www.w3.org/TR/coga-user-research/

### Incorrect use of H2 

![Screen Shot 2020-09-14 at 6 31 52 PM](https://user-images.githubusercontent.com/57469/93144878-10d50500-f6b9-11ea-92a2-7c753855098f.png)

<hr/>

### Correct use of p with .va-introtext

![Screen Shot 2020-09-14 at 6 31 29 PM](https://user-images.githubusercontent.com/57469/93144870-0c105100-f6b9-11ea-8ea3-ca70744016ed.png)

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

**Comment by bradl3yC** [comment]
@jenstrickland this should be in staging with the next deploy ~30 minutes or so - please validate at your leisure

**Comment by jenstrickland** [comment]
Validated
